catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -195,52 +237,47 @@ public LibraryImpl getLibrary() {
         return library;
     }
 
-    private ORB getOrbForServer(int port)
+    private void initCorba()
             throws InvalidName, AdapterInactive, WrongPolicy, ServantNotActive, IOException,
             SecurityServiceException {
 
-        System.setProperty("com.sun.CORBA.POA.ORBPersistentServerPort", String.valueOf(port));
-        System.setProperty("com.sun.CORBA.ORBServerPort", String.valueOf(port));
-
-        final ORB orb = ORB.init(new String[0], null);
-
-        System.clearProperty("com.sun.CORBA.POA.ORBPersistentServerPort");
-        System.clearProperty("com.sun.CORBA.ORBServerPort");
-
-        POA rootPOA = POAHelper.narrow(orb.resolve_initial_references("RootPOA"));
+        rootPOA = POAHelper.narrow(orb.resolve_initial_references("RootPOA"));
 
         rootPOA.the_POAManager()
                 .activate();
 
         library = new LibraryImpl(rootPOA);
         library.setCatalogFramework(framework);
-        Subject guestSubject = getGuestSubject();
-        library.setGuestSubject(guestSubject);
+        if (securityManager != null) {
+            Subject guestSubject = getGuestSubject();
+            library.setGuestSubject(guestSubject);
+        }
         library.setFilterBuilder(filterBuilder);
         library.setEnterpriseSearch(enterpriseSearch);
         library.setDefaultUpdateFrequencyMsec(defaultUpdateFrequencySec * 1000);
         library.setMaxPendingResults(maxPendingResults);
 
-        org.omg.CORBA.Object objref = rootPOA.servant_to_reference(library);
-
-        iorString = orb.object_to_string(objref);
+        libraryRef = rootPOA.servant_to_reference(library);
 
-        return orb;
+        iorString = orb.object_to_string(libraryRef);
```

Comment:
maybe wrap this in isDebugEnabled since this string is only used in a debug statement